target/linux/ath79/image/generic-tp-link.mk

```diff
@@ -191,6 +191,34 @@ define Device/tplink_archer-d50-v1
 endef
 TARGET_DEVICES += tplink_archer-d50-v1
 
+define Device/tplink_archer-d7-v1
+  $(Device/tplink-qca-dsl)
+  ATH_SOC := qca9558
+  DEVICE_MODEL := Archer D7
+  DEVICE_VARIANT := v1
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb-ledtrig-usbport kmod-ath10k-ct ath10k-firmware-qca988x-ct
```

Comment:
Please remove kmod-usb-core here and for the other device below, it is pulled automatically with kmod-usb2.

Reply:
Fixed! 😉